.circleci/config.yml

```diff
@@ -60,12 +60,20 @@ jobs:
       - run:
           name: e2e test
           command: |
+            set +e
             if [[ $CIRCLE_BRANCH != *"master"* || ! -z $CIRCLE_TAG ]] ; then
               E2EPROFILE=saucelight npm run test:e2e
             else
               npm run test:e2e
             fi
-  
+            STATUS=$?;
+            if [[ ! -z $FORCE_BUILD ]] ; then
+              echo 'FORCE_BUILD active!'
```

Comment:
can we elaborate on this message?

Reply:
Of course. :) But if the user uses the force build, she/he should know what is this. That's why I thought it can be enough. But it's done.